### PR TITLE
Resolves #4 user path for Chrome

### LINKED ISSUE #4: Not all chrome paths supported
On windows, my chrome.exe is located at 

`C:\Users\{USERNAME}\AppData\Local\Google\Chrome\Application\chrome.exe`

I'm looking into seeing if it's possible to use the `exists` function with a regex and modify [this block](https://github.com/ianmarmour/nvidia-clerk/blob/master/internal/browser/browser.go#L40) to also check for this path.

### PR DESCRIPTION
**Pull request by NickOttrando**
Resolves #4

### CODE REVIEW

**Comment by NickOttrando** [comment]
@ianmarmour please review. This was where my chrome was installed (newest version on fresh windows install) and fixed the issue for me. Tested it by running the application and no longer got the error about unable to find chrome.

**Comment by NickOttrando** [comment]
May also be a good idea to let user set their chrome location as env var.

**Comment by ianmarmour** [comment]
> @ianmarmour please review. This was where my chrome was installed (newest version on fresh windows install) and fixed the issue for me. Tested it by running the application and no longer got the error about unable to find chrome.

Will do I'm taking a look now.

**Comment by ianmarmour** [comment]
> May also be a good idea to let user set their chrome location as env var.

I agree, I'm trying to keep the number of env variables limited but it might make sense to have it be an optional override in case users have other non standard configurations.

File: internal/browser/browser.go [hunk 1]
```diff
@@ -117,7 +126,7 @@ func StartChromeDebugMode() bool {
 	case "windows":
 		cmd = exec.Command(getWindowsChromeLocation(), "--remote-debugging-port=9222", "--user-data-dir=remote-profile")
 	case "darwin":
-		cmd = exec.Command("/Applications/Google Chrome.app/Contents/MacOS/Google Chrome", "--remote-debugging-port=9222", "--user-data-dir=remote-profile")
+		cmd = exec.Command("open", "-a", "/Applications/Google \\Chrome.app", "--args", "--remote-debugging-port=9222", "--user-data-dir=remote-profile")
```
Comment (ianmarmour) [inline internal/browser/browser.go]: We were originally using this however it doesn't look like open -a properly supports --args for Chrome. The move to launching dthe app directly looks to have solved it, if you're running OSX can you verify this (I had a tested verify this change a few hours ago and the new strategy was working for him)

File: internal/browser/browser.go [hunk 2]
```diff
@@ -41,10 +41,19 @@ func getWindowsChromeLocation() string {
 		return "C:/Program Files (x86)/Google/Chrome/Application/chrome.exe"
```
Comment (ianmarmour) [inline internal/browser/browser.go]: I'm pretty unhappy with how I added all this code in to begin with, we probably want to refactor the function to return a string or an err and then do 

if check loc 1
  return loc1, nil
if check loc 2
 return loc2, nil
if check loc 3
 return loc3, nil

log.fatal(foobar)
return nil, err

**Review by ianmarmour — CHANGES_REQUESTED** [review]
Overall logic looks fine, lets refactor the function at the same time to clean it up I rushed this really badly to try and resolve some users complaints.

File: internal/browser/browser.go (hunk 2 above)
Comment (NickOttrando) [reply internal/browser/browser.go]: Sure, will update it tomorrow morning.

**Comment by NickOttrando** [comment]
@ianmarmour updated, please take a look. Should the binary be updated in the PR? I didn't add support for env var passing for now, but if someone else has an issue with their chrome being in another dir we can add pretty easily.

**Comment by ianmarmour** [comment]
@NickOttrando I actually need to go fix the github flows for different types of requests I smashed together this whole repo really quickly. There should be separate flows that trigger for PR's from build/releases (plus the release logic is broken too and should only be publishing drafts)

**Review by ianmarmour — APPROVED** [review]
Looks great to me approved.

**Comment by NickOttrando** [comment]
Happy to help, thank you!

**Pull request merged**